docs/source/about_dataset_load.mdx

```diff
@@ -0,0 +1,97 @@
+# Build and load
+
+Nearly every deep learning workflow begins with loading a dataset, which makes it one of the most important steps. With 🤗 Datasets, there are more than 900 datasets available to help you get started with your NLP task. All you have to do is call: [`datasets.load_dataset`] to take your first step. This function is a true workhorse in every sense because it builds and loads every dataset you use.
+
+## ELI5: `load_dataset`
+
+Let's begin with a basic Explain Like I'm Five.
+
+For community datasets, [`datasets.load_dataset`] downloads and imports the dataset loading script associated with the requested dataset from the Hugging Face Hub. The Hub is a central repository where all the Hugging Face datasets and models are stored. Code in the loading script defines the dataset information (description, features, URL to the original files, etc.), and tells 🤗 Datasets how to generate and display examples from it.
+
+If you are working with a canonical dataset, [`datasets.load_dataset`] downloads and imports the dataset loading script from GitHub.
+
+<Tip>
+
+Read the [Share](./share) section to learn more about the difference between community and canonical datasets. This section also provides a step-by-step guide on how to write your own dataset loading script!
+
+</Tip>
+
+The loading script downloads the dataset files from the original URL, generates the dataset and caches it in an Arrow table on your drive. If you've downloaded the dataset before, then 🤗 Datasets will reload it from the cache to save you the trouble of downloading it again.
+
+Now that you have a high-level understanding about how datasets are built, let's take a closer look at the nuts and bolts of how all this works.
+
+## Building a dataset
+
+When you load a dataset for the first time, 🤗 Datasets takes the raw data file and builds it into a table of rows and typed columns. There are two main classes responsible for building a dataset: [`datasets.BuilderConfig`] and [`datasets.DatasetBuilder`].
+
+
+<div class="flex justify-center">
+   <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/builderconfig.png"/>
+</div>
+
+### [`datasets.BuilderConfig`]
```

Comment:
https://github.com/huggingface/datasets/pull/3690/commits/c1bf3efec107cea5465ead219c0f49bedfb7c69a  Since `datasets.BuilderConfig` link is already the first word of the paragraph, I've removed the link from the heading (as related to https://github.com/huggingface/datasets/pull/3690#discussion_r811150659)
<img width="500" alt="Screenshot 2022-02-21 at 14 49 30" src="https://user-images.githubusercontent.com/11827707/154967733-b4c5ee6b-0178-49eb-84d3-580246cfefec.png">